frontend/assets/stylesheets/base/_tables.scss

```diff
@@ -55,3 +55,86 @@
         background-color: darken($c-neutral8, 4%);
     }
 }
+
+/* Comparison Table
+   ========================================================================== */
+
+$_table-tier-height: 150px;
```

Comment:
Yep, it's mostly a magic number, for the height of the tier-headers on small screens.